### DISCUSSION TITLE
Save and load grid functions and compute errors

### DISCUSSION THREAD

**Discussion by gokhalen** [mfem/mfem#3553] [General]
I'd like to save a couple of grid functions and then compute the L2 norm of the difference between them. How would I do this? I looked at `ComputeLpError()` in the class GridFunction but I'm not certain that this is what I want. GridFunction has a Save method to save to a stream, but there is not a Load method, it seems. 

Thanks, 

-Nachiket

**Comment by tzanio** [comment]
The counterpart to `Load` is just the constructor `GridFunction(Mesh *m, std::istream &input);`

Once you load two grid functions, you can wrap one as `GridFunctionCoefficient` and call `ComputeLpError()`. (Alternatively you can compute the difference and `ComputeLpError()` with a zero coefficient)

Hope this helps,
Tzanio
